pkg/cmd/run.go

```diff
@@ -53,18 +56,13 @@ getenvoy run ./envoy -- --config-path ./bootstrap.yaml
 
 # List available Envoy flags.
 getenvoy run standard:1.11.1 -- --help
+
+# Run with Postgres specific configuration bootstrapped
+getenvoy run postgres:nightly --templateArg Endpoint=127.0.0.0 --templateArg InPort=1114
```

Comment:
It's unexpected to see argument names starting from a capital letter: `Endpoint` and `InPort`.

I think, this is an implementation detail leaking into user interface.

I think, users should provide `endpoint` and `ingress.port`.